webapp/src/pages/Food.js

```diff
@@ -1,16 +1,73 @@
+import api from "../api";
 import foodImage from "../assets/images/onboarding-food.jpg";
-import WaitingListPage from "../components/WaitingListPage";
-import { mdp, t } from "../localization";
+import AppNav from "../components/AppNav";
+import PageLoader from "../components/PageLoader";
+import RLink from "../components/RLink";
+import { dayjs } from "../modules/dayConfig";
+import useAsyncFetch from "../shared/react/useAsyncFetch";
+import { LayoutContainer } from "../state/withLayout";
 import React from "react";
+import { Stack } from "react-bootstrap";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
 
 export default function Food() {
+  const { state: foodOfferings, loading: listLoading } = useAsyncFetch(
+    api.getFoodOfferings,
+    {
+      default: {},
+      pickData: true,
+    }
+  );
   return (
-    <WaitingListPage
-      feature="food"
-      imgSrc={foodImage}
-      imgAlt="Fresh produce in a market"
-      title={t("food:page_title")}
-      text={mdp("food:intro_md")}
-    />
+    <>
+      <AppNav />
+      <img src={foodImage} alt="food" className="thin-header-image" />
+      {listLoading ? (
+        <PageLoader />
+      ) : (
+        <LayoutContainer top>
+          <Row>
+            <h5 className="page-header mb-4">Vendor Offerings</h5>
+            {foodOfferings.map((o) => (
+              <Col xs={12} key={o.vendorId} className="mb-4">
+                <Card>
```

Comment:
Pull this card into its own component for this page. It will need further changes and is too nested to work with comfortably.